### PR TITLE
style: popover and command

### PR DESCRIPTION
**Pull request by Aztriltus**
## Changes
- Style popover
- Style command
- Make sure it's responsive

## Screenshots
https://github.com/AfterClass-io/afterclass.io-v2/assets/6003064/274c0184-d514-4cf9-8354-bf386b9dadb9

### CODE REVIEW

File: src/common/components/Popover/Popover.theme.ts [hunk 1]
```diff
@@ -5,16 +5,19 @@ export type PopoverVariants = VariantProps<typeof popoverTheme>;
...
+      "w-screen", // Required to make popover full width on mobile
+      "max-w-[320px]",
+      "min-w-0",
+      "rounded-xl",
+      "py-5",
+      "px-5",
       "border",
-      "p-4",
-      "shadow-md",
+      "border-border-base",
```
Comment (DavidLHW) [inline src/common/components/Popover/Popover.theme.ts:17]: issue:
don't think `border-border-base` exists, did you mean [`border-border-default`](https://github.com/AfterClass-io/afterclass.io-v2/blob/60-featui-popover-and-command-styling/src/common/tools/tailwind/themes/appTheme.ts#L32-L37)?

```suggestion
      "border-border-default",
```

image

**Review by DavidLHW — COMMENTED** [review]
thanks for the PR! 🔥 looks great - just 1 minor comment

safe to merge after addressing merge conflicts and the comment :)

File: src/common/components/Popover/Popover.theme.ts (hunk 1 above)
Comment (Aztriltus) [reply src/common/components/Popover/Popover.theme.ts]: Oh right. Nice catch. Fixed it here https://github.com/AfterClass-io/afterclass.io-v2/pull/62/commits/38926eb13ce03403f889b6c8016f3f270fc83695

**Review by DavidLHW — APPROVED** [review]
CLGTM @ 1e4f6a1! 🚀

**Pull request merged**